projects/plugins/jetpack/extensions/blocks/dialogue/attributes.js

```diff
@@ -1,15 +1,21 @@
 export default {
-	participantSlug: {
-		type: 'string',
+	participant: {
+		type: 'object',
 	},
 	timestamp: {
 		type: 'string',
 		default: '00:00',
 	},
+	showTimestamp: {
+		type: 'boolean',
+		default: false,
+	},
 	placeholder: {
 		type: 'string',
 	},
 	content: {
 		type: 'string',
+		source: 'html',
+		selector: '.wp-block-jetpack-dialogue__content',
```

Comment:
This puts the dialogue content in the html markup, within a `<p>` so it's displayed.